airbyte-integrations/connectors/source-opsgenie/unit_tests/test_stream.py

```diff
@@ -0,0 +1,508 @@
+#
+# Copyright (c) 2022 Airbyte, Inc., all rights reserved.
+#
+
+import unittest
+
+import responses
+from source_opsgenie.streams import AlertLogs, AlertRecipients, Alerts, Incidents, Integrations, Services, Teams, Users, UserTeams
```

Comment:
`Alerts` stream, as per comments

Reply:
Reintroduced alerts testing. No need to remove.